theano/sandbox/multinomial.py

```diff
@@ -219,6 +219,17 @@ class ChoiceFromUniform(MultinomialFromUniform):
 
     """
 
+    __props__ = ("replace",)
+
+    def __init__(self, replace=False, *args, **kwargs):
```

Comment:
It don't have the same param order as the GPU version. I would go the same as the GPU version:

def __init__(self, odtype, replace=False):

then when you call super, you just pass odtype, the only one needed to pass.